src/components/vertical-menu/responsive-vertical-menu/responsive-vertical-menu.component.tsx

```diff
@@ -41,6 +43,7 @@ export interface ResponsiveVerticalMenuProps extends TagProps {
 const BaseMenu = ({
   children,
   height,
+  launcherButtonAriaLabel = "Main menu",
```

Comment:
suggestion (non-blocking): I wonder if it would be more appropriate not to surface a prop here as this is not being applied to the root element, we have an aria-label for the close button as well and it's highly unlikely there'll be more than one instance of the component on a given page. My thinking is we can use the translations to handle this and provide localisation at the same time for both element's aria-label attribute

Reply:
comment: Either way I think it's worth adding tests for this so we can catch any regressions at an early stage etc